src/components/tx/TxData.js

```diff
@@ -156,7 +163,10 @@ class TxData extends React.Component {
    * @param {Object} e Event emitted when clicking link
    */
   toggleRaw = e => {
-    e.preventDefault();
+    if (e) {
```

Comment:
chore: Every call made to this method has the event object. I believe this conditional is not necessary.

Reply:
This conditional was added to avoid the <a> navigation behavior in the old UI. It is also checked as a parameter to not try to use preventDefault in the DropDetails onT function. We now update that inside DropDetails it uses a parameterless arrow function to pull the function, but still maintaining a conditional
. Please Review. 🙂